setup.py

```diff
@@ -457,6 +468,8 @@ def _read_requirements(filename: str) -> List[str]:
 
 if _is_cuda() or _is_hip():
     ext_modules.append(CMakeExtension(name="vllm._moe_C"))
+    ext_modules.append(
+        CMakeExtension(name="vllm.vllm_flash_attn.vllm_flash_attn_c"))
```

Comment:
This might only be applicable in CUDA mode.  I don't know if it has ever been tested with HIP/AMD